tests/TestCase/Filesystem/FolderTest.php

```diff
@@ -680,23 +680,50 @@ public function testSlashTerm()
     /**
      * testNormalizePath method
      *
+     * @group deprecated
      * @return void
      */
     public function testNormalizePath()
     {
         $path = '/path/to/file';
+        /** @noinspection PhpDeprecationInspection */
         $result = Folder::normalizePath($path);
         $this->assertEquals('/', $result);
 
         $path = '\\path\\\to\\\file';
+        /** @noinspection PhpDeprecationInspection */
         $result = Folder::normalizePath($path);
         $this->assertEquals('/', $result);
 
         $path = 'C:\\path\\to\\file';
+        /** @noinspection PhpDeprecationInspection */
```

Comment:
You can use the `$this->deprecated()` helper to swallow deprecation warnings during this test. We generally don't use these comment directives either.